rfcs/text/0002-rfc-environment.md

```diff
@@ -138,29 +185,27 @@ The goal here is to research and understand the impact of these changes on users
 
 ## Concerns
 
- * Beats (Filebeat, Metricbeat, Heartbeat...) document in their configuration files (filebeat.yml...) an alternate non standardizable field `fields.env` illustrated with the demo value `staging`.
+<!--
+Stage 1: Identify potential concerns, implementation challenges, or complexity. Spend some time on this. Play devil's advocate. Try to identify the sort of non-obvious challenges that tend to surface later. The goal here is to surface risks early, allow everyone the time to work through them, and ultimately document resolution for posterity's sake.
+-->
+
+ * Beats (Filebeat, Metricbeat, Heartbeat...) document in their configuration files (filebeat.yml, metricbeat.yml...).
```

Comment:
A format that's been working well in other RFCs is to capture concerns in a format of concern + resolution.

Since some of the concerns are complex (multi-line, code snippets, etc) I don't think the bullet points work well. Perhaps you could separate each concern with a `###` heading instead? This way you're free to have paragraphs and examples as needed. You can check out how I've done it [in this RFC](https://github.com/elastic/ecs/blob/master/rfcs/text/0007-multiple-users.md#concerns) for an example.

Reply:
Problem solved